gdcdictionary/schemas/observation.yaml

```diff
@@ -773,3 +773,19 @@ properties:
 
   subjects:
     $ref: "_definitions.yaml#/to_one"
+
+# Prop originated for Longcovid and Monkeypox data
+
+  observations_icd10:
+    description: >
+      zero, one or more ICD-10 codes for encounter
+    type: array
+      items:
+        type: string
+
+  observations_cpt:
+      description: >
+        zero, one or more CPT codes for encounter
+      type: array
+        items:
+          type: string
```

Comment:
i think the indentation here is causing the tests failures

Reply:
I am sorry about that. I have updated the code and made commit.